cloudify_rest_client/ldap.py

```diff
@@ -0,0 +1,83 @@
+########
```

Comment:
maybe ldapclient.py is a better name than ldap.py, since you expose LdapClient..